site/role/manifests/device.pp

```diff
@@ -0,0 +1,10 @@
+class role::device {
+
+  # This role includes profiles for each aspect of a device
+  include profile::banner
+  include profile::network_vlan
+  include profile::interface
+  include profile::ntp_server
+  include profile::commands
```

Comment:
The commands are Cisco IOS specific, so it might be confusing to suggest they work on a generic 'device'